packages/office-ui-fabric-react/src/utilities/dateMath/DateMath.ts

```diff
@@ -150,9 +150,17 @@ export function compareDatePart(date1: Date, date2: Date): Number {
  * @param {DateRangeType} dateRangeType - The desired date range type, i.e., day, week, month, etc.
  * @param {DayOfWeek} firstDayOfWeek - The first day of the week.
  * @param {DayOfWeek[]} workWeekDays - The allowed days in work week. If not provided, assumes all days are allowed.
+ * @param {number} daysToSelectInDayView - The number of days to include when using dateRangeType === DateRangeType.Day
+ * for multiday view. Defaults to 1
  * @returns {Date[]} An array of dates representing the date range containing the specified date.
  */
-export function getDateRangeArray(date: Date, dateRangeType: DateRangeType, firstDayOfWeek: DayOfWeek, workWeekDays?: DayOfWeek[]): Date[] {
+export function getDateRangeArray(
+  date: Date,
+  dateRangeType: DateRangeType,
+  firstDayOfWeek: DayOfWeek,
+  workWeekDays?: DayOfWeek[],
+  daysToSelectInDayView: number = 1
```

Comment:
>daysToSelectInDayView [](start = 2, length = 21)

what happens when this value < 1?

Reply:
> what happens when daysToSelectInDayView < 1?

I tried it... it's bad. Definately should be `Math.max(daysToSelectInDayView, 1)`